AutoHub.DAL/EntitySettings/LotConfiguration.cs

```diff
@@ -10,6 +10,8 @@ public LotConfiguration(EntityTypeBuilder<Lot> entity)
         {
             entity.ToTable("Lot").HasKey(lot => lot.LotId);
 
+            entity.Navigation(lot => lot.LotStatus).AutoInclude();
```

Comment:
Use the same approach of including everywhere. If you use a custom generic include method, then do not use AutoInclude()